src/util/TickUtils.ts

```diff
@@ -152,10 +201,65 @@ function getTicksStart(
   return result.filter(entry => entry.isShow);
 }
 
-// todo Array<Tick>
-// TODO: Step 1 - Add unit tests. Document existing behaviour
-// Step 2 - add unit tests for equidistant behaviour.
-// Do it.
+function getEquidistantTicks({
```

Comment:
I'm not sure I understand what is happening here 😂

Reply:
I initially copied the getTicksStart function and appended some extra logic. Now instead I reuse it.